src/pybind/mgr/influx/module.py

```diff
@@ -49,6 +53,14 @@ class Module(MgrModule):
                 'name': 'verify_ssl',
                 'default': 'true'
             },
+            {
+                'name': 'threads',
+                'default': 5
+            },
+            {
+                'name': 'batch_size',
+                'default': 5000
```

Comment:
What kind of advice would we give people about how to choose this value?  I wonder if we should just hard-code it to avoid yet another hard-to-explain config value.

Reply:
As I said, 1.000 to 10.000 is the recommended value according to InfluxDB.